blog/2019/05/19/new-site.md

```diff
@@ -0,0 +1,34 @@
+---
+title: New site engine for rexify.org
+date: 2019-05-19 00:00:00
+tags: news
+---
```

Comment:
I guess we could use `author` in frontmatter too.

Reply:
Added.